R/oauth-init.R

```diff
@@ -62,12 +62,14 @@ init_oauth1.0 <- function(endpoint, app, permission = NULL,
 #'     retrieve the token. Some authorization servers require this.
 #'     If \code{FALSE}, the default, retrieve the token by including the
 #'     app key and secret in the request body.
+#' @param config_init Additional configuration settings sent to
+#' \code{\link{POST}}, e.g. \code{\link{user_agent}}.
```

Comment:
Needs to be indented - http://style.tidyverse.org/code-documentation.html#indention